### ISSUE TITLE
[PDKI-655] PentANON: Signature detection - improvement

### ISSUE THREAD

**Issue by jinijohn26** [5Plus-Test/jiramigrationtest#415]
> Migrated from Jira: [PDKI-655](https://pentadoc.atlassian.net/browse/PDKI-655)

HQ: *I-0049 - KI Team* => PentANON

---
### Checklist

- [x] have an initial meeting to define who does what of the different to dos @712020:bba6203e-8aaa-4243-bf57-36e87a20749e create a label studio export @712020:b8b0c8e1-2fcb-4591-a2da-64666dadf710 go through the export and pick the documents that have signatures in them and add into the training set (with a python script) wait for results from the combined model @712020:b8b0c8e1-2fcb-4591-a2da-64666dadf710 @712020:bba6203e-8aaa-4243-bf57-36e87a20749e finetune parameters @712020:b8b0c8e1-2fcb-4591-a2da-64666dadf710 @712020:bba6203e-8aaa-4243-bf57-36e87a20749e  run final training

---
### Comments

**Naveed Ahmad** (02/Jan/25 11:01 AM):
> Sent a meeting invite to BOL, VIT and JIJ
